server/auth/routes.js

```diff
@@ -88,4 +88,26 @@ router.post("/register", async (req, res, next) => {
     });
 });
 
+//Documents created
+
+router.post("/", async (req, res, next) => {
```

Comment:
What is this endpoint? Should it be removed as your API endpoint is implemented in api/documents.js ?

Reply:
yes it should be removed, this was done by mistake